POINTS.md

```diff
@@ -5,49 +5,31 @@ To be considered as co-author, 10 contribution points is required.
 > **Note**: The purpose of the point system is not to barrier collaboration, but to reward rare and high-quality dataset entries.
 We might adjust the point requirement lower to accomodate more co-authorship, if needed.
 
-## Data Loader
+## Implementing Data Loader
 
-Implementing any data loader is granted +3 pts.
+Implementing any data loader is granted +3 pts, unless otherwise specified on the Github issue.
 More details [here](DATALOADER.md).
 
-## Dataset Proposal
+## NusaCatalogue's Datasheet Proposal
 
-Point from dataset proposal depends on various factors:
+### Datasheet Proposal as the Dataset's Author(s)
+Proposing a datasheet of your own dataset is granted +4 pts.
 
-### Size
+As a support for data openness, for any data that is previously private, if Author(s) agree to make the dataset publicly available then additional +2 pts will be granted.
 
-We can have 4 different levels: Small, Medium, Large, XL
+As a support for the development of local languages datasets:
+- For dataset in Sundanese, Javanese, or Minangkabau, +2 pts will be granted
+- For dataset in other local language, +4 pts will be granted
 
-- Small (S): <1K (+1 pts)
-- Medium (M): 1K<=x<10K (+ 2pts)
-- Large (L): 10K<=x<1M (+3 pts)
-- Extra Large (XL): >=1M (+4 pts)
+We understand that the quality of a dataset varies a lot. To support fairness in scoring datasets with different quality, for any dataset that does not achieve a certain minimum standard, 50% contribution score of the dataset will be penalized. This policy affects dataset that is collected with:
+- Crawling without any manual validation process
+- Machine / heuristic-rule labelled dataset, without any manual validation
+- Machine translated dataset without any manual validation.
 
-where x is the number of samples (train + val + test). For the dataset with k-fold evaluation setting, we will only count one of the folds.
+> **Note**: if there is more than 1 Author for a dataset, main Author will be eligible for nominating 1 more Author to be granted the same contribution score.
 
-### Task and Language Rarity
+### Datasheet Proposal not as the Dataset's Author(s)
+Proposing a datasheet of other people dataset is granted +1 pt. Contribution score will be granted after the original Author agrees to put the dataset in NusaCatalogue.
```

Comment:
maybe we should give +1 point regardless? even if the original author don't agree?

Reply:
Yeah, I was also thinking the same thing before. Ok, let me update it now.

Reply:
Updated